openedx_webhooks/tasks/github.py

```diff
@@ -55,15 +58,18 @@ def pull_request_opened(pull_request, ignore_internal=True, check_contractor=Tru
                 repo=repo, num=num,
             ),
         )
-        comment = {
-            "body": github_internal_cover_letter(pr),
-        }
-        url = "/repos/{repo}/issues/{num}/comments".format(
-            repo=repo, num=num,
-        )
+        coverletter = github_internal_cover_letter(pr),
```

Comment:
Can you update the docstring to match the new behavior (GitHub won't let me tag it directly). It looks like it has been wrong for a while as it says it ignores internal pull requests.